plugins/view-resources/src/components/Table.svelte

```diff
@@ -37,14 +37,18 @@
   let objects: Doc[]
 
   const q = createQuery()
-  $: q.query(
-    _class,
-    query,
-    (result) => {
-      objects = result
-    },
-    { sort: { [sortKey]: sortOrder }, ...options, limit: 200 }
-  )
+
+  async function update(_class: Ref<Class<Doc>>, query: DocumentQuery<Doc>, sortKey: string, sortOrder: SortingOrder, options: FindOptions<Doc> | undefined) {
```

Comment:
```suggestion
  async function update(_class: Ref<Class<Doc>>, query: DocumentQuery<Doc>, sortKey: string, sortOrder: SortingOrder, options?: FindOptions<Doc>) {
```

Is function is better?